UdpHosts/GameServer/StaticDB/Records/dbcharacter/Faction.cs

```diff
@@ -1,6 +1,10 @@
+using System.Collections.Generic;
+
 namespace GameServer.Data.SDB.Records.dbcharacter;
 public record class Faction
 {
+    public static readonly HashSet<uint> HostileFactionIds = new() { 2, 3, 5, 6, 7, 8, 17, 22, 42, 43, 45, 46, 47, 48 };
```

Comment:
I'd like to see this moved somewhere else, maybe into HardcodedCharacterData.
My primary concern in this case is I want to keep the non customdb classes clean for possible reuse in other projects that want to use the SDB.